src/plugins/kibana/public/visualize/wizard/step_2.html

```diff
@@ -1,37 +1,26 @@
-<h1>
-  Select a search source
-  <span class="pull-right label label-default hidden-xs">Step 2</span>
-  <span class="label label-default visible-xs">Step 2</span>
-</h1>
-<ul class="list-group list-group-menu" ng-switch on="stepTwoMode">
-  <li ng-click="stepTwoMode='new'"
-      ng-class="{'active': stepTwoMode=='new'}"
-      class="list-group-item list-group-menu-item">
-    From a new search
-  </li>
-  <li class="list-group-item" ng-switch-when="new">
+<div class="wizard">
+  <div class="wizard-column">
+    <h3>From a New Search</h3>
     <!-- Index patterns -->
-    Select an index pattern
-    <div class="form-group">
-      <select
-              class="form-control"
-              ng-model="indexPattern.selection"
-              ng-options="pattern as pattern for pattern in indexPattern.list | orderBy:'toString()'">
-      </select>
+    <div class="wizard-row">
+      <div class="panel panel-default">
+        <div class="panel-heading">Index Patterns</div>
+      </div>
+      <ul class="striped list-group">
+        <li class="list-group-item" ng-repeat="pattern in indexPattern.list">
```

Comment:
This list should be paginated like the saved object loader.

Also, in all of the other columns, the entire row is clickable. In this one, only the name of the pattern is clickable.